content/articles/understanding-docker-networking/index.md

```diff
@@ -0,0 +1,81 @@
+Containers are a popular way of deploying applications quickly and reliably. Containers run in a set of servers or a platform. For containers to communicate with each other, it is vital to use docker networking. Docker takes care of all networking aspects so that containers can talk with each other and the Docker host.
+
+Docker has a networking model that allows containers to communicate automatically. Also, it enables communication to the same or different hosts and the outside world. This article will give an understanding of how to manage connections between containers.
+
+In the current world, containerization is popular among enterprises. Thus good networking skills are needed. Networking is a significant component of container deployment and management. This article will review the networking aspects of Docker.
+
+### Table of contents
+[Docker Networking](#docker-networking)
+[Why Docker Networking is Important](#why-docker-networking-is-important)
+[Container Network Models](#container-network-models)
+[Network Drivers](#network-drivers)
+
+### Prerequisites
+A comprehensive understanding of [Docker](https://www.section.io/engineering-education/docker-concepts/) is needed to follow along with this article. Also, go through [networking overview](https://docs.docker.com/network/) on Docker documentation.
+
+### Docker networking
+Networking involves communication among processes. Networking in Docker is no different. Docker networking initiate communication between Docker containers and the outside world via the [host machine](https://docs.docker.com/network/host/). Docker utilizes a container network model (CNM) for networking. This blueprint regulates the steps needed to provide networking for containers using drivers.
```

Comment:
Could you elaborate more on this sentence `Networking involves communication among processes`